tests/BUILD

```diff
@@ -1113,6 +1113,11 @@ py_test(
     ],
 )
 
+jax_test(
```

Comment:
This could be py_test if it isn't hardware-dependent.

Reply:
(with some additional deps)

Reply:
Done (finally)